src/components/StoriesGroup/StoriesGroup.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import {Modal, ModalCloseReason} from '../Modal';
+import {IndexType, StoriesLayout} from '../Stories/components/StoriesLayout/StoriesLayout';
+import {StoriesPreview} from './components';
+import {StoriesGroupItem} from './types';
+
+import {block} from '../utils/cn';
+import './StoriesGroup.scss';
+
+const b = block('stories-group');
+
+export interface StoriesGroupProps {
+    open: boolean;
+    groups: StoriesGroupItem[];
+    initialStoryIndex?: [groupIndex: number, itemIndex: number];
+    disableOutsideClick?: boolean;
+    onClose?: (
+        event: MouseEvent | KeyboardEvent | React.MouseEvent<HTMLElement, MouseEvent>,
+        reason: ModalCloseReason | 'closeButtonClick',
+    ) => void;
+    onItemSelect?: (
+        itemIndexes: [groupIndex: number, itemIndex: number],
+        selectedFromThumbnail?: boolean,
```

Comment:
Lets make it required argument

Reply:
[37ebf42](https://github.com/gravity-ui/uikit/pull/387/commits/37ebf42123aef15bb3632269ddde6af48e838464)

Reply:
Nope, I've meant itemIndexes and selectedFromThumbnail both required, not both optional